Lib/fontTools/feaLib/ast.py

```diff
@@ -117,14 +109,17 @@ def asFea(g):
 
 
 class Element(object):
+    """A base class representing "something" in a feature file."""
 
     def __init__(self, location=None):
+        #: location of this element - tuple of ``(filename, line, column)``
```

Comment:
Does this format mean that the field `location` of `Element` is documented but that the parameter `location` of `Element.__init__` is not documented? Or does Sphinx figure it out and use the same docstring for both?

For this particular field, the docstring is omitting the possibility that the location is `None`.

Reply:
`#:` is indeed a special comment in sphinx